scripts/preparepackages.js

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const { Listr } = require( 'listr2' );
+const releaseTools = require( '@ckeditor/ckeditor5-dev-release-tools' );
+const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
+
+const latestVersion = releaseTools.getLastFromChangelog();
+const versionChangelog = releaseTools.getChangesForVersion( latestVersion );
+
+const tasks = new Listr( [
+	{
+		title: 'Verifying the repository.',
+		task: async () => {
+			const errors = await releaseTools.validateRepositoryToRelease( {
+				version: latestVersion,
+				changes: versionChangelog,
+				branch: 'master'
+			} );
+
+			if ( !errors.length ) {
+				return;
+			}
+
+			return Promise.reject( 'Aborted due to errors.\n' + errors.map( message => `* ${ message }` ).join( '\n' ) );
+		}
+	},
+	{
+		title: 'Running build command.',
+		task: () => {
+			return tools.shExec( 'yarn run build', { async: true, verbosity: 'silent' } );
+		}
+	},
+	{
+		title: 'Updating the `#version` field.',
+		task: () => {
+			return releaseTools.updateVersions( {
+				version: latestVersion
+			} );
+		}
+	},
```

Comment:
```suggestion
	{
		title: 'Updating the `#version` field.',
		task: () => {
			return releaseTools.updateVersions( {
				version: latestVersion
			} );
		}
	},
	{
		title: 'Running build command.',
		task: () => {
			return tools.shExec( 'yarn run build', { async: true, verbosity: 'silent' } );
		}
	},
```

I wonder if we shouldn't update the version before building the package. It's a rare case, but if the `build` task load `package.json#version` field anywhere in the `dist/` code, it will point to the previous version.

Reply:
Yeah, I agree that this order is safer.